website/blog/2023-07-16-databend-weekly-102.md

```diff
@@ -0,0 +1,171 @@
+---
+title: "This Week in Databend #102"
+date: 2023-07-16
+slug: 2023-07-16-databend-weekly
+cover_url: 'weekly/weekly-102.png'
+image: 'weekly/weekly-102.png'
+tags: [weekly]
+description: "Stay up to date with the latest weekly developments on Databend!"
+contributors:
+  - name: ariesdevil
+  - name: b41sh
+  - name: BohuTANG
+  - name: Chasen-Zhang
+  - name: dantengsky
+  - name: Dousir9
+  - name: everpcpc
+  - name: flaneur2020
+  - name: JackTan25
+  - name: lichuang
+  - name: PsiACE
+  - name: RinChanNOWWW
+  - name: SkyFan2002
+  - name: soyeric128
+  - name: sundy-li
+  - name: TCeason
+  - name: xudong963
+  - name: youngsofun
+  - name: zhang2014
+  - name: zhyass
+authors:
+  - name: PsiACE
+    url: https://github.com/psiace
+    image_url: https://github.com/psiace.png
+---
+
+[Databend](https://github.com/datafuselabs/databend) is a modern cloud data warehouse, serving your massive-scale analytics needs at low cost and complexity. Open source alternative to Snowflake. Also available in the cloud: <https://app.databend.com> .
+
+## What's On In Databend
+
+Stay connected with the latest news about Databend.
+
+### Add Bloom Index to the Specified Column
+
+Creating a bloom index consumes a lot of CPU resources. For wide tables where only a few columns may require point queries or data ingestion performance is more important, creating bloom indexes for all columns may not be a good idea.
+
+Databend [Fuse Engine](https://databend.rs/doc/sql-reference/table-engines/fuse) added support for the `bloom_index_columns` option, which means you can add bloom indexes to specified columns as needed.
+
+To create table with bloom index:
+
+```SQL
+CREATE TABLE table_name (
+  column_name1 column_type1,
+  column_name2 column_type2,
+  ...
+) ... bloom_index_columns='columnName1[, ...]'.
+```
+
+To create or modify bloom index for a existing table:
+
+<small><i>The existing Bloom index options will be replaced by the new options, it does not create Bloom filters for existing data.</i></small>
+
+```SQL
+ALTER TABLE <db.table_name> SET OPTIONS(bloom_index_columns='columnName1[, ...]');
+```
+
+To disable the bloom index:
+
+```SQL
+ALTER TABLE <db.table_name> SET OPTIONS(bloom_index_columns='');
+```
+
+If you are interested in learning more, please check out the resources listed below.
+
+- [PR #12048 | feat: support specify bloom index columns](https://github.com/datafuselabs/databend/pull/12048)
+
+### Understand SQL Conformance
+
+Databend aims to conform to the SQL standard, with particular support for ISO/IEC 9075:2011, also known as SQL:2011. Databend incorporates many features required by the SQL standard, often with slight differences in syntax or function.
+
+We have added a summary about the level of conformity of Databend to the SQL:2011 standard, hoping it can help you further understand Databend's SQL Conformance.
```

Comment:
```suggestion
We have summarized the level of conformity of Databend to the SQL:2011 standard, hoping it can help you further understand Databend's SQL Conformance.
```